src/components/Console.jsx

```diff
@@ -9,6 +16,10 @@ import {EditorLocation} from '../records';
 import ConsoleEntry from './ConsoleEntry';
 import ConsoleInput from './ConsoleInput';
 
+library.add(faBan);
+library.add(faChevronDown);
+library.add(faChevronUp);
```

Comment:
@chasestarr my apologies! i should have been clearer about what i was proposing. i think the simplest/clearest thing might be to avoid the `library` concept altogether and just directly use the imported icons in the JSX, e.g.:

```js
import {
  faBan,
  faChevronDown,
  faChevronUp,
} from '@fortawesome/free-solid-svg-icons';
import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';

// …

<FontAwesomeIcon icon={ban} />
```

This follows the [explicit import](https://github.com/FortAwesome/react-fontawesome#explicit-import) approach mentioned in their documentation.

Would that work?

Reply:
👍 yes that is much better